.github/workflows/ci-docker.yml

```diff
@@ -0,0 +1,48 @@
+name: pressio-containers
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'develop'
+  pull_request:
+    types: [ opened, reopened, synchronize ]
+    paths:
+      - 'docker_scripts/**'
+  schedule:
+    - cron:  '0 5 * * FRI'
+
+jobs:
+  CI:
+    strategy:
+      matrix:
+        config:
+          - {dockerfile: 'fedora-clang',         tag: 'latest'}
+          - {dockerfile: 'fedora-gnu',           tag: 'latest'}
+          # - {dockerfile: 'intel-oneapi',         tag: 'latest'}
```

Comment:
Intel's repository GPG key is expired and `intel-oneapi` image fails to build. There might be some workaround for this, but I'd rather deal with it in a follow-up PR (since intel build is disabled in `ci-baseline.yml` anyways).

Reply:
yes no need for now